apps/src/code-studio/pd/workshop_dashboard/components/survey_results_header.jsx

```diff
@@ -331,7 +331,9 @@ const SurveyResultsHeader = React.createClass({
   },
 
   getCoursesForWorkshops() {
-    return _.uniq(_.flatMap(this.props.workshops, workshop => workshop.course));
+    let courses = _.uniq(_.flatMap(this.props.workshops, workshop => workshop.course));
+    _.remove(courses, (course) => {return ['Admin', 'Counselor'].indexOf(course) >= 0;});
+    return courses;
```

Comment:
Maybe just use some es6 functions? Specifically the `Set` class, `Array.from`, `Array.filter` and `Array.includes`:

```jsx
const courses = Array.from(new Set(this.props.workshops.map(workshop => workshop.course)));
return courses.filter(course => ['Admin', 'Counselor'].includes(course));
```